### PR TITLE
Run more tests on both JVM/JS

### PR DESCRIPTION
**Pull request by armanbilge**
fs2-io supports all APIs (including files) on JVM and JS now.

### CODE REVIEW

**Comment by armanbilge** [comment]
All the sources are shared now, so you can simplify your directory structure to an ordinary project (no need for shared/jvm/js split).

File: build.sbt
Comment (armanbilge) [inline build.sbt:283]: This setting controls the directory structure.

**Comment by armanbilge** [comment]
@satabin should I fix these modules as well?
https://github.com/satabin/fs2-data/blob/bf5a7033574d17c05cf408ef33a93b7792ec1d8a/build.sbt#L250-L251
https://github.com/satabin/fs2-data/blob/bf5a7033574d17c05cf408ef33a93b7792ec1d8a/build.sbt#L264-L265

**Comment by satabin** [comment]
Ok thanks I need to do a global check on my use of `shared`,  I thought I had more specific cases.

**Comment by ybasket** [comment]
I let two of you figure out the sbt sources setup, it's by far not my strength 🙄 

But thank you very much for the work @armanbilge! Feel free to ping me on the Typelevel Discord (same handle) if you need more workflow approvals on the way.

File: build.sbt
Comment (ybasket) [inline build.sbt:60]:
```suggestion
    "io.circe" %%% "circe-jawn" % circeVersion % "test",
```

Or is there any reason we need the milestone instead of the stable `0.14.1` that's used elsewhere?

Comment (armanbilge) [reply build.sbt:60]: Yes, to get the circe-jawn parser on JS. It will be available in 0.14.2 when that's released.

Comment (satabin) [reply build.sbt:60]: I’m fine with that since it’s for testing purpose.

**Review by ybasket — APPROVED** [review]
Looks good from my side then, but I'll let @satabin do the merge in case there's something with the source folders still to be optimised.

**Comment by satabin** [comment]
> Looks good from my side then, but I'll let @satabin do the merge in case there's something with the source folders still to be optimised.

Nah, it’s fine, I’ll merge it. Thanks a lot @armanbilge!

**Pull request merged**
